### PR TITLE
GSREN3D-151: Navigation buttons

### PR DESCRIPTION
**Pull request by akhelouat**
GSREN3D-151

### JIRA issue

https://jira.camptocamp.com/browse/GSREN3D-151

### Description

Adding navigation buttons
- return to home
- zoom in 
- zoom out

### Screenshots

image

### CODE REVIEW

File: src/components/map/MapComponent.vue [hunk 1]
```diff
@@ -43,7 +71,12 @@ layerStore.$subscribe(() => {
   <div class="absolute right-2 top-2 z-10">
     <TransportButtons></TransportButtons>
   </div>
-  <div class="absolute right-2 bottom-2">
-    <UiButton @click="toggleMap">{{ is3D() ? '2D' : '3D' }}</UiButton>
+  <div class="absolute right-4 bottom-2 flex flex-col [&>*]:m-2 text-[#5E5E5E]">
```
Comment (ismailsunni) [inline src/components/map/MapComponent.vue:74]: better not to use custom color here. It's better to put declare it on tailwind.config.cjs

**Review by ismailsunni — COMMENTED** [review]
You need also to make sure the linter pass

File: src/components/map/NavigationButtons.vue [hunk 2]
```diff
@@ -0,0 +1,64 @@
...
+
+  await activeMap?.gotoViewpoint(homeViewPoint)
+}
+</script>
+
+<template>
+  <div class="absolute right-4 bottom-2 flex flex-col [&>*]:m-2 text-gray-dark">
+    <UiButton @click="returnToHome"><IconHome /></UiButton>
+    <div
+      class="flex flex-col zoom-buttons text-2xl [&>*]:p-2 first:[&>*]:rounded-b-3xl last:[&>*]:rounded-t-3xl"
```
Comment (ismailsunni) [inline src/components/map/NavigationButtons.vue:55]: what does this [&>*] stuff? Sorry, I am not familiar, and when google it gives me a completely different thing.

**Review by ismailsunni — APPROVED** [review]
LGTM

File: src/components/map/NavigationButtons.vue (hunk 2 above)
Comment (akhelouat) [reply src/components/map/NavigationButtons.vue:55]: it's for targeting the child elements.

**Pull request merged**